app/controllers/todos_controller.rb

```diff
@@ -0,0 +1,5 @@
+class TodosController < ApplicationController
+	def index
```

Comment:
Check your indentation here; sublime might be misconfigured. It looks too-indented to my eyes, but it might be github.